server/docs/source/index.html.md

```diff
@@ -1267,6 +1293,92 @@ For example `"secret_name": "FOO_DOMAIN_COM"` will write following secrets to th
 
 `POST /v1/certificates/{grid_id}/certificate`
 
+# Volumes
+
+**Volumes support is at experimental state which means that there might be breaking changes between versions until the experimental status is removed.**
+
+## Volume
+
+```json
+{
+  "id": "my-grid/foo",
+  "name": "foo",
+  "scope":"instance",
+  "driver":"local",
+  "driver_opts": {
+    "driver_specific_option": "foobar",
+    "another_option": "xyz"
+  },
+  "instances": [
+    {
+      "name":"stack.svc.foo-1",
+      "node": "node-1"
+    }
+  ],
+  "services": [
+    {
+      "id":"my-grid/stack/svc"
+    }
+  ]
+}
+```
+
+Attribute | Description
+--------- | -----------
+name      | Name of the volume
+scope     | Scope for the volume (`instance`, `stack` or `grid`)
+driver    | Volume driver to be used. Each node reports it's supported drivers, see [node details](#get-a-node-details)
+driver_opts| Options for the volume driver
+
+## List volumes
+
+Lists volumes created to a grid
+
+```http
+GET /v1/volumes/{grid_id} HTTP/1.1
+Authorization: bearer 8dqAd30DRrzzhJzbcSCG0Lb35csy5w0oNeT+8eDh4q2/NTeK3CmwMHuH4axcaxya+aNfSy1XMsqHP/NsTNy6mg==
+Accept: application/json
+```
+
+### Endpoint
+
+`GET /v1/volumes/{grid_id}`
```

Comment:
I find the `GET /v1/volumes/{grid_id}` vs `GET /v1/volumes/:grid/:volume` surprising, but this is consistent with the other API endpoint docs.